hyperspy/signal.py

```diff
@@ -4641,12 +4642,13 @@ def map(
         function,
         show_progressbar=None,
         parallel=None,
-        max_workers=None,
         inplace=True,
         ragged=None,
         output_signal_size=None,
         output_dtype=None,
-        **kwargs
+        lazy_result=None,
+        max_workers=None,
```

Comment:
Move it back to its original place to avoid changing order of argument - which is not necessary here.